front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -117,6 +129,49 @@ export default function CreateAssistant({
     );
   };
 
+  const tagsRefsMap = useRef<{
+    [key: string]: React.MutableRefObject<HTMLDivElement | null>;
+  }>({});
+
+  useEffect(() => {
+    templateTags.forEach((tag: string) => {
+      tagsRefsMap.current[tag] = tagsRefsMap.current[tag] || createRef();
+    });
+  }, [templateTags]);
+
+  const scrollToTag = (tagName: string) => {
```

Comment:
Any reason for not keeping the anchor behavior provided by NextJS 🙃?

Reply:
We wanted a smooth animation taking into account the header + a shake animation. 
If it's doable with NextJs anchor system happy to change that.